packages/Python/lldbsuite/test/lang/swift/completion/TestSwiftREPLCompletion.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import print_function
+import pexpect
+import os
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class TestSwiftREPLCompletion(TestBase):
+    mydir = TestBase.compute_mydir(__file__)
+
+    @skipUnlessDarwin
+    def test_repl_completion(self):
+        prompt = "Welcome to"
+        child = pexpect.spawn('%s --repl' % (lldbtest_config.lldbExec))
+        # Assign to make sure the sessions are killed during teardown
+        self.child = child
+        # Send a <TAB> and make sure we don't crash.
+        child.sendline("import Foundation\t")
```

Comment:
what about sending
print(NSString("ok"))
afterwards and expecting "ok"?

Reply:
Does Linux Foundation have NSString?

Reply:
Good point.